controllers/users.js

```diff
@@ -196,18 +196,16 @@ const postUserPicture = async (req, res) => {
  */
 const updateUser = async (req, res) => {
   try {
-    const userId = req.params.userId;
-    const { id: profileDiffId, message, ...profileDiff } = req.body;
+    const { id: profileDiffId, message } = req.body;
+
+    const { approval, timestamp, userId, ...profileDiff } = await profileDiffsQuery.fetchProfileDiff(profileDiffId);
+    if (!profileDiff) return res.boom.notFound("Profile Diff doesn't exist");
```

Comment:
Does this throw `notFound` error when we pass a random profileDiffId? Was it tested?

Reply:
This function doesn't check for that. It only checks if profileDiff is empty the throws an error.

Reply:
So can we do something like this? 
```js
    const profileDiffData = await profileDiffsQuery.fetchProfileDiff(profileDiffId);
    if (!profileDiffData) return res.boom.notFound("Profile Diff doesn't exist");
    
    const { approval, timestamp, userId, ...profileDiff } = profileDiffData;
```

Reply:
What will be the advantage of this change?

Reply:
Try passing a random profileDiffId and compare the result.

Reply:
Got it 👍